src/tests/shopping_cart.spec.js

```diff
@@ -0,0 +1,31 @@
+const { expect } = require('@playwright/test');
+const deepEqual = require('deep-equal');
+const { test } = require('../fixture');
+const { randomChoiceItems } = require('../utils/choice_random_items');
+const { standardUser } = require('../../config/credentials');
+
+test.describe('Adding random items to the shopping cart', () => {
+    test('Add and verify items', async ({ inventoryPage, loginPage, baseSwagLabPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin(standardUser.username, standardUser.password);
+
+        const allItems = await inventoryPage.inventoryItems.count();
+        const randomItems = randomChoiceItems(allItems);
+
+        await inventoryPage.addItemsToCart(randomItems);
+
+        const cartItemsNames = await inventoryPage.getInventoryItemsNames(randomItems);
+        const cartItemsDescriptions = await inventoryPage.getInventoryItemsDescriptions(randomItems);
+        const cartItemsPrices = await inventoryPage.getInventoryItemsPrices(randomItems);
+
+        await baseSwagLabPage.shoppingCart.click();
+
+        const shoppingCartItemsNames = await inventoryPage.getInventoryItemsAllNames();
+        const shoppingCartItemsDescriptions = await inventoryPage.getInventoryItemsAllDescriptions();
+        const shoppingCartItemsPrices = await inventoryPage.getInventoryItemsAllPrices();
+
+        deepEqual(cartItemsNames, shoppingCartItemsNames);
```

Comment:
https://playwright.dev/docs/api/class-genericassertions#generic-assertions-to-equal could we use this assert from the Playwright?

Reply:
`expect(cartItemsNames).toEqual(shoppingCartItemsNames);`

I used to have this before, but hlazkov wrote a comment - "for arrays its better to use deep equal"

Reply:
In our particular case, https://playwright.dev/docs/api/class-genericassertions#generic-assertions-to-equal will be enough. But feel free to leave it.